### DISCUSSION TITLE
Linux(Ubuntu 22.04) DesktopGL project using system's default icon

### DISCUSSION THREAD

**Discussion by WoodardEric** [MonoGame/MonoGame#8293] [Desktop computers]
I followed this fix to change my games icons https://github.com/MonoGame/MonoGame/issues/8035, and it worked great on Windows. On Linux it would only use Ubuntu's default gear icon. So then I created a new project called Test using the .NET template, and I found that it is also using Ubuntu's default icon.

![Screenshot from 2024-05-01 13-49-06](https://github.com/MonoGame/MonoGame/assets/47343793/72832dfa-36e8-4922-a100-51cb852e26b6)

However the task bar icon does work.

![Screenshot from 2024-05-01 17-27-51](https://github.com/MonoGame/MonoGame/assets/47343793/b606895d-e1c9-4f6f-af65-f59a67c0f335)

I'm thinking this is a issue with MonoGame or .NET but I mostly use Linux with the terminal so I figured I would see what over people thought before making an issue. To recreate run `dotnet new mgdesktopgl -o MyGame` on a Ubuntu PC. Then build or publish the project and look at the resulting executable from the bin folder.

**Comment by JupiterRider** [comment]
It is not possible to set an icon for the Linux executable. This has nothing to do with Monogame or .NET. 
On Linux you would use .desktop files for that.
